airflow/contrib/hooks/qubole_hook.py

```diff
@@ -207,6 +210,30 @@ def get_jobs_id(self, ti):
             cmd_id = ti.xcom_pull(key="qbol_cmd_id", task_ids=self.task_id)
         Command.get_jobs_id(self.cls, cmd_id)
 
+    def get_extra_links(self, task, dttm):
+        log = LoggingMixin().log
```

Comment:
is there a reason we put the log as local variable not a global variable?

Reply:
Removed.